eskip/eskip.go

```diff
@@ -96,6 +96,21 @@ type Route struct {
 	Backend string
 }
 
+type RoutePredicate func(*Route) bool
```

Comment:
As long as we use this type somewhere, I think a type alias like this one makes code more readable (even, or especially, for simple types like this)